pilot/pkg/features/pilot.go

```diff
@@ -157,6 +157,13 @@ var (
 		false,
 		"Use the Istio JWT filter for JWT token verification.")
 
+	// SkipValidateTrustDomain tells the server proxy to not to check the peer's trust domain when
+	// mTLS is enabled in authentication policy.
+	SkipValidateTrustDomain = env.RegisterBoolVar(
+		"SKIP_VALIDATE_TRUST_DOMAIN",
```

Comment:
To keep consistency, suggest adding `PILOT_` prefix

Reply:
Done.